content/docs/gke/troubleshooting-gke.md

```diff
@@ -15,14 +15,25 @@ This guide covers troubleshooting specifically for
 For more help, try the 
 [general Kubeflow troubleshooting guide](/docs/other-guides/troubleshooting).
 
-## Troubleshooting Cloud Identity-Aware Proxy (Cloud IAP)
+## Troubleshooting kubeflow deployment on GCP
 
 Here are some tips for troubleshooting Cloud IAP.
 
- * Make sure you are using HTTPS.
- * See the guide to 
+* Make sure you are GCP project owner
```

Comment:
Add a full stop at the end of the line, for consistency with other bullet points.